apps/src/music/utils/Notes.ts

```diff
@@ -45,11 +45,15 @@ export const getNoteOctave = (note: number): number =>
 export const getPitchName = (note: number): string =>
   getNoteName(note) + getNoteOctave(note);
 
-// Transpose the note by adding the note offset to the target note defined
-// by the target key.
-export const getTranposedNote = (targetKey: Key, noteOffset: number) =>
+// Transpose a note from relative to absolute, by adding the note offset
+// to the target note defined by the target key.
+export const getTransposedNote = (targetKey: Key, noteOffset: number) =>
   targetKey + ROOT_NOTE_START + noteOffset;
 
+// Untranspose a note from absolute to relative.
+export const getUntransposedNote = (targetKey: Key, note: number) =>
```

Comment:
Maybe worth renaming this pair of functions as something like `getAbsolutePitch/getRelativePitch` or `absoluteToRelativePitch/relativeToAbsolutePitch`? Technically speaking "transposing" just means changing the key, so "untransposing" isn't really the operation here (I guess that might refer to transposing back to the original key?)

Reply:
Good call.  Done.